plugins/print/plugin.js

```diff
@@ -36,12 +36,10 @@
 	 * type and should be compatible with its API.
```

Comment:
Could we mention here that `exportpdf` plugin is required and link to guide how to set it up?